tests/ftests/076-pybindings-cgroup_get_procs.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgroup_get_procs() test using the python bindings
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli
+from libcgroup import Cgroup, Version
+from process import Process
+import consts
+import ftests
+import sys
+import os
+
+
+CGNAME = '076getprocs/cgwithpids'
+EMPTY_CGNAME = '076getprocs/cgwithoutpids'
+CONTROLLERS = ['cpu', 'pids']
+PID_CNT = 20
+
+initial_pid_list = list()
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    return result, cause
+
+
+def setup(config):
+    global initial_pid_list
+    CgroupCli.create(config, CONTROLLERS, CGNAME)
+    CgroupCli.create(config, CONTROLLERS, EMPTY_CGNAME)
+
+    for i in range(0, PID_CNT):
+        pid = config.process.create_process(config)
+        initial_pid_list.append(pid)
+
+    CgroupCli.classify(config, CONTROLLERS, CGNAME, initial_pid_list, ignore_systemd=True)
+    initial_pid_list = initial_pid_list.sort()
```

Comment:
@drakenclimber patch looks good. A few minor nitpicks, `cgroup_get_procs()` does a sort before returning the pid list, do you see an unsorted pid list?

Reply:
I have seen the list be in a different order coming back from `cgroup_get_procs()`, and this caused failures as I was writing the test.  Not sure why it was out of order, honestly.

It's likely not needed after calling `create_process()` as the PIDs coming back from it should be monotonically increasing, but I don't see a downside to doing it.

Reply:
Agreed, there is not down side to sorting it. I was just curious.